packages/suite/src/utils/suite/router.ts

```diff
@@ -139,5 +78,6 @@ export const isInternalRoute = (route: string) => {
 };
 
 export const isStatic = (route: string) => {
-    return routesConstants.staticRoutes.includes(route);
+    const routeFound = routes.find(r => r.pattern === route);
+    return routeFound ? !!routeFound.isStatic : true; // 404 page act as a static
```

Comment:
why do you need !! ? dont say it doesnt work, but seems not to be needed.

Reply:
beucase isStatic can be undefined, and this awesome func should return boolean to make sense not boolean | undefined.

Reply:
ah so, makes sense now.